camp/jbolotski/beam_break_motor/src/main/java/frc/robot/ManualInputInterfaces.java

```diff
@@ -0,0 +1,31 @@
+// ************************************************************
+// Bishop Blanchet Robotics
+// Home of the Cybears
+// FRC - Rapid React - 2022
+// File: ManualInputInterfaces.java
+// Intent: Forms a class that grants access to driver controlled inputs.
+// ************************************************************
+
+// ʕ •ᴥ•ʔ ʕ•ᴥ•  ʔ ʕ  •ᴥ•ʔ ʕ •`ᴥ´•ʔ ʕ° •° ʔ ʕ •ᴥ•ʔ ʕ•ᴥ•  ʔ ʕ  •ᴥ•ʔ ʕ •`ᴥ´•ʔ ʕ° •° ʔ 
+
+package frc.robot;
```

Comment:
So I think in this situation (where the motor always runs and their is a single command only ever going to run), I'd likely model this beam break sensor as a subsystem. 

If instead you were to flip around and treat the beam break sensor change as an event to schedule a command, then it seems like an events type class (or maybe even a folder where we keep all 'event-type' objects) would be in order.

Reply:
Looks like there is a beam break class in the subsystems.

Reply:
Keep looking below to 'Trigger' discussion

Reply:
done.